out_of_tree_boards/boards/arm/mec172x_ptl_uh/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2024 Microchip Technology Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#Allow users to pre-specify the tool using '-DMEC172X_SPI_GEN=<path-to-tool>/toolname'
+if (NOT DEFINED MEC172X_SPI_GEN)
+  set(MEC172X_SPI_GEN $ENV{MEC172X_SPI_GEN})
+endif()
+find_program (MEC172X_SPI_GEN NAMES mec172x_spi_gen_lin_x86_64 mec172x_spi_gen)
+if(MEC172X_SPI_GEN STREQUAL MEC172X_SPI_GEN-NOTFOUND)
+  message(WARNING "Microchip SPI Image Generation tool is not available. SPI Image will not be generated.")
+endif()
+
+if (NOT DEFINED MEC172X_SPI_CFG)
+  set(MEC172X_SPI_CFG $ENV{MEC172X_SPI_CFG})
+endif()
+find_file (MEC172X_SPI_CFG NAMES spi_cfg_4MBit.txt spi_cfg_128MBit.txt spi_cfg.txt PATHS ${BOARD_DIR}/support)
+if(MEC172X_SPI_CFG STREQUAL MEC172X_SPI_CFG-NOTFOUND)
+  message(WARNING "Microchip SPI Image Generation tool configuration file is not available. SPI Image will not be generated.")
+endif()
+
+  if(DEFINED ENV{SPI_IMAGE_NAME})
+    set(SPI_IMAGE_NAME $ENV{SPI_IMAGE_NAME})
+  else()
+    set(SPI_IMAGE_NAME ECFW.bin)
```

Comment:
Set this ksc.bin until we update the build.sh later on 
Refer to https://github.com/intel/ecfw-zephyr/blob/master/.ci-functions/build.sh#L64